project/plugins.sbt

```diff
@@ -18,8 +18,9 @@ addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.0") // for advan
 addSbtPlugin("io.spray" % "sbt-boilerplate" % "0.6.1")
 addSbtPlugin("org.foundweekends" % "sbt-bintray" % "0.5.2")
 addSbtPlugin("com.lightbend.sbt" % "sbt-javaagent" % "0.1.4")
+addSbtPlugin("com.lightbend.paradox" % "sbt-paradox" % "0.6.1")
+addSbtPlugin("com.lightbend.paradox" % "sbt-paradox-apidoc" % "0.2")
 addSbtPlugin("com.lightbend.akka" % "sbt-paradox-akka" % "0.18")
```

Comment:
This one is up to 0.20 now but side note: https://github.com/akka/akka/pull/27473#issuecomment-519795226

Reply:
Yep, we should do that in another step and visually check the change.